symfony/framework-bundle/5.3/config/services.yaml

```diff
@@ -0,0 +1,24 @@
+# This file is the entry point to configure your own services.
+# Files in the packages/ subdirectory configure your dependencies.
+
+# See https://symfony.com/doc/current/best_practices.html#configuration
+parameters:
+
+services:
+    # default configuration for services in *this* file
+    _defaults:
+        autowire: true      # Automatically injects dependencies in your services.
+        autoconfigure: true # Automatically registers your services as commands, event subscribers, etc.
+
+    # makes classes in src/ available to be used as services
+    # this creates a service per class whose id is the fully-qualified class name
+    App\:
+        resource: '../src/'
+        exclude:
+            - '../src/DependencyInjection/'
+            - '../src/Entity/'
+            - '../src/Kernel.php'
+            - '../src/Tests/'
+
+    # add more service definitions when explicit configuration is needed
```

Comment:
No more autodiscovery for the `App\Controller\` namespace, and not more `src/Controller/.gitignore` file in the recipe.

Reply:
This means controllers will only be discovered automatically if they extend the `AbstractController` class, right?

Reply:
Yes!

Reply:
I suppose that requires updating the documentation first then.

Reply:
It *is* an ugly spot in the config file. But yes, we would definitely need to cover this in the docs - this would be a big WTF possibly.

Reply:
I don't understand this change. We used this sine Symfony 4.4:

```yaml
    # controllers are imported separately to make sure services can be injected
    # as action arguments even if you don't extend any base controller class
    App\Controller\:
        resource: '../src/Controller/'
        tags: ['controller.service_arguments']
```

It worked beautifully all these years. If there isn't a strong technical reason to change this, we should keep it. Thanks!

Reply:
IIRC, this was added to support controllers not using the `AbstractController` as base class

Reply:
@stof is correct. In my opinion, this is supporting an uncommon use case, which we don't have to by default. Better keep config files lean.

Reply:
It's OK then if we consider that it's an uncommon use case.

But please, let's try to minimize changes because it hurts docs maintenance and it's a pain for some users 🙏  Thanks!